api/proto/teleport/access_graph/v1/secrets_service.proto

```diff
@@ -18,23 +18,40 @@ package teleport.access_graph.v1;
 
 import "teleport/access_graph/v1/authorized_key.proto";
 import "teleport/access_graph/v1/private_key.proto";
-import "teleport/devicetrust/v1/device_collected_data.proto";
-import "teleport/devicetrust/v1/devicetrust_service.proto";
+import "teleport/devicetrust/v1/assert.proto";
 
 option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/accessgraph/v1;accessgraphv1";
 
-// SecretsScannerService provides methods for Access Graph Secret Scanner
-// functionality.
+// SecretsScannerService provides methods for Access Graph Secret Scanner functionality.
 service SecretsScannerService {
   // ReportAuthorizedKeys is used by Teleport SSH nodes to report authorized keys
   // that could be used to bypass Teleport.
+  // The client (Teleport SSH Node) should authenticate using the certificate-key pair signed by Teleport HostCA.
   rpc ReportAuthorizedKeys(stream ReportAuthorizedKeysRequest) returns (stream ReportAuthorizedKeysResponse) {}
+
   // ReportSecrets is used by trusted devices to report secrets found on the host
   // that could be used to bypass Teleport.
+  // The client (device) should first authenticate using the [ReportSecretsRequest.device_assertion] flow.
+  //
+  // Assertion ceremony flow:
+  // -> AssertDeviceInit (client)
+  // <- AssertDeviceChallenge (server)
+  // -> AssertDeviceChallengeResponse (client)
+  // <- DeviceAsserted (server)
```

Comment:
Just refer to AssertDeviceRequest and/or lib/devicetrust/assert and let the documentation be there. This is likely to become outdated or incomplete over time.